llvm/lib/Target/AArch64/AArch64ISelDAGToDAG.cpp

```diff
@@ -326,9 +326,14 @@ class AArch64DAGToDAGISel : public SelectionDAGISel {
     return false;
   }
 
-  template <unsigned BaseReg> bool ImmToTile(SDValue N, SDValue &Imm) {
+  template <unsigned BaseReg, unsigned Max>
+  bool ImmToTile(SDValue N, SDValue &Imm) {
```

Comment:
If `ImmToTile` is now used for ZT, this is no longer the right name. Can you rename this to `ImmToReg` or something that like that instead?